tests/AsyncBridge.Tests/PartialTrustTests.cs

```diff
@@ -0,0 +1,273 @@
+﻿using System;
+using System.Reflection;
+using System.Runtime.Serialization;
+using System.Security;
+using System.Security.Permissions;
+using System.Security.Policy;
+using System.Threading;
+using System.Threading.Tasks;
+using Microsoft.VisualStudio.TestTools.UnitTesting;
+#if NET45
+using TaskEx = System.Threading.Tasks.Task;
+#endif
+
+#if NET45
+namespace ReferenceAsync.Tests
+#elif NET35
+namespace AsyncBridge.Net35.Tests
+#elif ATP
+namespace AsyncTargetingPack.Tests
+#else
+namespace AsyncBridge.Tests
+#endif
+{
+#if !ATP
+    public sealed class Sandbox : IDisposable
+    {
+        private readonly AppDomain _Sandbox;
+
+        public Sandbox()
+        {
+            var CurrentSetup = AppDomain.CurrentDomain.SetupInformation;
```

Comment:
Adding https://github.com/OmerMor/AsyncBridge/issues/19 asap. Sorry one more time! 😟
Can you use camel-case for both fields and locals, optionally with underscore for fields?